scripts/tools/nrfconnect/generate_nrfconnect_chip_factory_data.py

```diff
@@ -99,8 +99,6 @@ def gen_test_certs(chip_cert_exe: str,
                    vendor_id: int,
                    product_id: int,
                    device_name: str,
-                   generate_cd: bool = False,
```

Comment:
I'm not sure that we can remove that, because some customers can use it to generate their CD. That option doesn't include CD in the FD set, but it generates CD to the build directory.